entries/p/powerreviews.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "PowerReviews": {
+    "domain": "powerreviews.com",
```

Comment:
Their `https://powerreviews.com` certificate isn't valid.
```suggestion
    "domain": "powerreviews.com",
    "url": "https://www.powerreviews.com"
```

Reply:
<img width="349" alt="3" src="https://user-images.githubusercontent.com/71176569/124931326-655bbf00-e002-11eb-9f0c-819b6109206a.png">

Reply:
do
```BASH
curl -I https://powerreviews.com/
```
Does it fail?

Reply:
You are right, the certificate isn't valid